### PR TITLE
Dashboard approvals

### PR DESCRIPTION
**Pull request by Michael-B-Lee**
Connects to #11

### CODE REVIEW

File: app/components/Dashboard.js [hunk 1]
```diff
@@ -1,112 +1,95 @@
...
+          daysOff: '6 days',
+          returnDate: 'Returning Wed 28/09/2016'
+        }
+      ]
+    };
+  },
+  render: function () {
+    var listRequests = this.state.Employees.map(function (obj, idx) {
+      return (
+      <div className="absenceRequest" key={idx}>
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: indentation

File: app/components/Settings.js [hunk 2]
```diff
@@ -1,13 +1,460 @@
...
+    );
+  }
+});
+
+  //Allowances Table
+  var AllowancesTable = React.createClass({
+    getInitialState: function () {
+      return {
+        teams:[
+        {
```
Comment (AlexJYoungman) [inline app/components/Settings.js]: Rename to allowances instead of teams. Just stops it being confusing

File: app/components/Settings.js [hunk 3]
```diff
@@ -1,13 +1,460 @@
...
+          <div className="panel panel-default">
+            <div className="panel-heading">Allowances</div>
+            <table>
+              <tr className="table-allowances">
+                <div className="mCellsContainer">
+                  <div className="mCellsNames">
+                    <tr>Annual Holiday Allowance</tr>
+                    <tr>Residual Holiday Allowance</tr>
+                    <tr>Charity Days Allowance</tr>
+                  </div>
```
Comment (AlexJYoungman) [inline app/components/Settings.js]: Should this not read from the state instead of being hardcoded?

File: app/components/Dashboard.js [hunk 4]
```diff
@@ -1,112 +1,95 @@
...
-              </div>
+              {listRequests}
             </div>
           </div>
         </div>
       </div>
-    );
-  }
-});
+      );
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: indentation

File: app/components/Dashboard.js [hunk 5]
```diff
@@ -141,7 +125,7 @@ var ApproveAbsence = React.createClass({
     var self = this;
     var teamsTable = this.state.absenceRequests.map(function (obj, idx) {
       return (
-        <table>
+        <table className="absenceRequestTable">
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: indentation

File: app/components/Dashboard.js [hunk 6]
```diff
@@ -194,7 +178,7 @@ var ApproveAbsence = React.createClass({
         <div className="col-md-4">
           <div className="panel panel-default">
             <div className="panel-heading">Absence Request</div>
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: Check the whole page for indentation, there are a couple I spotted around line 180 but cant view them on github

File: app/components/Dashboard.js [hunk 7]
```diff
@@ -150,7 +134,7 @@ var ApproveAbsence = React.createClass({
             <td className="mAbsCellsNames">{obj.user}</td>
             <tr className="reasonDate">
               <td className="mAbsCellsReason">{obj.absenceType}</td>
-              <td className="mAbsCellsDate">{obj.fromDate} &#45; {obj.toDate}</td>
+            <img src= "../app/images/bulletPointSeparator.png" id="icon-circle" />   <td className="mAbsCellsDate">{obj.fromDate}  {obj.toDate}</td>
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: dont have the dashes in the {/*------Hidden table rows that display when approve/decline absence request buttons are pressed------*/} comment

File: app/components/Dashboard.js [hunk 8]
```diff
@@ -297,87 +281,176 @@ var ApproveAbsence = React.createClass({
...
+    </div>
+
     );
   }
 });
 
+var PieChart = React.createClass({
+
+  piechart: function () {
+    var width = 600;                                          //sets the width of the chart
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: Try to put the comments above the variable like so... 

//sets the width of the chart
var width = 600;

File: app/styles/css/dashboard.css [hunk 9]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (TheGarrett) [inline app/styles/css/dashboard.css]: remove

**Pull request merged**